arelle/plugin/validate/NL/rules/nl_kvk.py

```diff
@@ -271,6 +271,33 @@ def rule_nl_kvk_3_2_1_1 (
         )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NL_INLINE_2024
+    ],
+)
+def rule_nl_kvk_3_2_3_1 (
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    NL-KVK.3.2.3.1: Transformation Registry 4 or newer are allowed. Everything else is prohibited.
+    """
+    transformRegistryErrors = []
+    for fact in val.modelXbrl.facts:
+        if fact is not None and fact.nsmap['ixt'] in DISALLOWED_IXT_NAMESPACES:
```

Comment:
The nsmap doesn’t indicate whether the namespace has been applied as a transformation to the fact. To verify this, use the `fact.format.namespaceURI` attribute instead.